cli/auth.go

```diff
@@ -49,10 +49,10 @@ const (
 	defaultOpenIDDiscoveryPath = "/.well-known/openid-configuration"
 
 	prodAuthDomain = "https://auth.viam.com"
-	prodAudience   = "https://app.viam.com/"
-	prodClientID   = "HysEkkRKn6cDr2W6UFI6UYJHpiVwXFCk" // native client ID
+	prodAudience   = "ce2227b8-4e43-46af-b701-a4b3359d4b4e"
+	prodClientID   = "ce2227b8-4e43-46af-b701-a4b3359d4b4e" // native client ID
 
-	stgAuthDomain = "https://auth.viam.dev/"
```

Comment:
specifically shouldn't have had a `/` at the end of the domain (we append `/.well-known/openid-configuration` to it)